cmd/kubeadm/app/preflight/checks_test.go

```diff
@@ -193,7 +193,9 @@ func TestFileExistingCheck(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Failed to create file: %v", err)
 	}
+	f.Close()
 	defer os.Remove(f.Name())
```

Comment:
Can we move the two lines in the same deferred block everywhere? That way we don't need to worry about if we call the f.Close() too early, or if the Close() and Remove() are called in the right order.
```suggestion
	defer func() {
	  f.Close()
	  os.Remove(f.Name())
	}()
```

Reply:
It's my initial thought,too.I will do that.